File: frontend/src/app/modules/arquivo/models/responses/get-arquivo-recepcionado-datatable-response.ts
import { GetArquivoRecepcionadoDatatableDto } from "../dtos/get-arquivo-recepcionado-datatable.dto";

export class GetArquivoRecepcionadoDatatableResponse {
    data?: GetArquivoRecepcionadoDatatableDto[];
    totalRecords?: number;
    page?:number;
    pageSize?:number;
}

File: backend/CaseTecnico.MRA.Application/UseCases/ArquivoRecepcionados/GetArquivoRecepcionadoDatatable/GetArquivoRecepcionadoDatatableResponse.cs

using CaseTecnico.MRA.Application.UseCases.Arquivos.GetArquivoDatatable;

namespace CaseTecnico.MRA.Application.UseCases.ArquivoRecepcionados.GetArquivoRecepcionadoDatatable;

public class GetArquivoRecepcionadoDatatableResponse
{
    public List<GetArquivoRecepcionadoDatatableDto> Data { get; set; } = new();

    // ----- Paginação -----
    public int TotalRecords { get; set; }
    public int Page { get; set; }
    public int PageSize { get; set; }
}
